content/kubernetes/re-clusters/multi-namespace.md

```diff
@@ -82,8 +86,8 @@ roleRef:
 Apply the files:
 
 ```sh
-kubectl apply -f role.yaml
-kubectl apply -f role_binding.yaml
+kubectl apply -f role.yaml -n <managed namespace>
```

Comment:
Although it's trivial for some, I think we typically add a comment such as "replace <managed namespace> with the namespace in which you wish to create an REDB", @kaitlynmichael wdyt?